src/Commands/Site/CreateCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Site;
+
+/**
+ * Class CreateCommand
+ * @package Pantheon\Terminus\Commands\Site
+ */
+class CreateCommand extends SiteCommand
+{
+    /**
+     * Creates A site
+     *
+     * @command site:create
+     *
+     * @usage terminus site:create <sitename> --label <label> --upstream <upstream>
+     *   Creates a site with the listed options.
+     *
+     * @param string $sitename Name of the site to create
+     * @param string $label Human readable label
+     * @param string $upstream UUID of the upstream product
+     * @option string $org UUID of Orginization to use
+     * @usage terminus site:create <site> <label> <upstream>
```

Comment:
Please add a line under this describing the anticipated behavior when using this command.